bundles/core/org.eclipse.smarthome.core.persistence/META-INF/MANIFEST.MF

```diff
@@ -6,8 +6,20 @@ Bundle-Version: 0.8.0.qualifier
 Bundle-ManifestVersion: 2
 Bundle-License: http://www.eclipse.org/legal/epl-v10.html
 Bundle-SymbolicName: org.eclipse.smarthome.core.persistence
-Import-Package: org.eclipse.smarthome.core.items,
+Import-Package: io.swagger.annotations,
+ javax.servlet,
+ javax.servlet.http,
+ javax.ws.rs,

```

Comment:
please make all rest related imports optional, so that the bundle is also usable without REST support and will then simply not activate the rest resource component.